.github/workflows/dco.yml

```diff
@@ -0,0 +1,18 @@
+name: Developer Certificate of Origin Check
+
+on: [pull_request]
```

Comment:
Can you change to only run on `main` then? For example, see lines 6-8 here: https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/blob/main/.github/workflows/unit-tests-workflow.yml#L6

Reply:
Just doubled checked with Peter regarding this. Seems like the standards needs to be defaulted to all branches. Reasoning given: "otherwise you could miss commits that were squashed before being prepped to merge with main. If accept pushes to main, then you should also run on main, but if there is a violation you will need to force update the branch (not desirable)"

Reply:
I'm still confused on why that would be necessary. Why would there be commits pushed to the base repo not going through `main` first? For example, there is not supposed to be feature branches in base repo at all; everything is supposed to get merged into `main`, then backported to their respective version branch..

I'm ok with this though, if it's the provided script from the issue.